onnx/defs/nn/defs.cc

```diff
@@ -2074,4 +2074,41 @@ ONNX_OPERATOR_SET_SCHEMA(
              {{"Processed_STD"}, "Add", {"STD", "Epsilon"}},
              {{"Y"}, "Div", {"X_variance", "Processed_STD"}}})));
 
+static const char* celu_ver12_doc = R"DOC(
+       Continuously Differentiable Exponential Linear Units:
+       Perform the linear unit element-wise on the input tensor X
+       using formula: <br/> ``` max(0,x) + min(0,α*(exp(x/α)−1)) ```
```

Comment:
Very nice `alpha` symbol but the attribute uses plain text name `alpha`... I am also afraid of that this cool symbol can't be displayed in my VIM.